exporter/clickhouselogsexporter/factory.go

```diff
@@ -38,7 +38,7 @@ func NewFactory() component.ExporterFactory {
 	return component.NewExporterFactory(
 		typeStr,
 		createDefaultConfig,
-		component.WithLogsExporterAndStabilityLevel(createLogsExporter, component.StabilityLevelBeta),
+		component.WithLogsExporter(createLogsExporter, component.StabilityLevelBeta),
```

Comment:
Any specific reason to set stability to beta?

Reply:
Not specific, just waiting to see if we can find any more issues before we declare it stable.